src/domain/users/index.types.d.ts

```diff
@@ -7,11 +7,29 @@ type UserLanguage =
 declare const deviceTokenSymbol: unique symbol
 type DeviceToken = number & { [deviceTokenSymbol]: never }
 
+type UserContact = {
+  id: string
+  name: string
+  transactionsCount: number
+}
+
+type QuizQuestion = {
+  id: string
+  earnAmount: number
+}
+
+type UserQuizQuestion = {
+  question: QuizQuestion
+  completed: boolean
+}
+
 type User = {
   id: UserId
   username: Username | null
   phone: PhoneNumber
   language: UserLanguage
+  contacts: UserContact[]
+  quizQuestions: UserQuizQuestion[]
```

Comment:
same here, to what wallet the rewards will be added? (if the user is in multiple accounts)

Reply:
Good question for when we support multiple wallets, but we'll need to convert the existing API (for now)